docs/development/build-instructions-gn.md

```diff
@@ -174,6 +174,16 @@ $ ./out/Debug/electron.exe
 $ ./out/Debug/electron
 ```
 
+### Packaging
+
+To strip the debugging and symbol information and package electron 
+as a distributable zip file:
+
+```sh
+electron/script/strip-binaries.py -d out/Release
```

Comment:
Can we note that this is for linux dist?

Reply:
Sure. But is there an equivalent step for Windows users I should mention?

Reply:
There isn't an equivalent step for Windows/Mac users.  Stripping is only necessary for linux.

Reply:
Ok, then the changes I did should be sufficient.